README.md

```diff
@@ -0,0 +1,45 @@
+# Tune Tracker
+
+## Problem
+[Spotify](https://open.spotify.com/) is a great source for DJs to find new records. However, getting copies of these tracks to be used in mixing sets is a manual task which includes but is not limited to the following steps:
+
+- Buying / Downloading tracks. Features idle time while waiting for a download to complete.
+- Arranging files by folders / labelling track files to simplify mxing in record box or another mixing program. 
+
+## Solution
+Tune Tracker (name is not final) is a web application that aims to solve this problem giving the users an opportunity to provide a spotify playlist URL to monitor for new tracks. Locating, purchasing and downloading of the music will be handled automatically.
+
+## Architecture
+
+TBD
+
+## Running the App Locally
```

Comment:
so i should follow this guide first, yahs ?

Reply:
Yep, will require some installation, but should work! 👍